packages/web/src/pages/trending-page/components/RewardsBanner.tsx

```diff
@@ -67,9 +67,10 @@ const RewardsBanner = ({ bannerType }: RewardsBannerProps) => {
       direction={isMobile ? 'column' : 'row'}
       alignItems='center'
       onClick={() => onClick(bannerType)}
+      pv='m'
+      ph='2xl'
       css={{
-        padding: '9px 34px',
-        background: color.special.gradient,
+        background: 'linear-gradient(315deg, #5B23E1 0%, #A22FEB 100%)',
```

Comment:
noticing we have an opp to drastically simply this container using Paper and enabling linear gradient background through theming, probably worth another sync on this